types/graphql-react/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true,
+        "jsx": "react"
+    },
+    "files": [
+        "index.d.ts",
+        "graphql-react-tests.ts"
```

Comment:
I changed this line to `"graphql-react-tests.tsx"`.

Reply:
🤦🏻‍♂️that's the part I was missing... I had tried creating both files, and simply importing the `.tsx` from the `.ts`. This, though, caused the linter to fail because it thought it was an "unnecessary file."